python/ray/tune/suggest/nevergrad.py

```diff
@@ -119,7 +137,7 @@ def on_trial_complete(self,
         """
         ng_trial_info = self._live_trial_mapping.pop(trial_id)
         if result:
-            self._nevergrad_opt.tell(ng_trial_info, -result[self._reward_attr])
+            self._nevergrad_opt.tell(ng_trial_info, self._metric_op * result[self._metric])
```

Comment:
careful, as `nevergrad.tell` automatically minimizes (rather than maximizes). You will need to update the switch case in L81 to reflect this.